app/src/main/java/org/oppia/app/databinding/DrawableBindingAdapters.kt

```diff
@@ -13,3 +13,10 @@ fun setBackgroundDrawable(view: View, @ColorInt colorRgb: Int) {
   // The input color needs to have alpha channel prepended to it.
   (view.background as GradientDrawable).setColor((0xff000000 or colorRgb.toLong()).toInt())
 }
+
+@BindingAdapter("app:rectangleDrawableWithColor")
+fun setRectangleBackgroundDrawable(view: View, @ColorInt colorRgb: Int) {
+  view.setBackgroundResource(R.drawable.rectangle_background)
```

Comment:
I think this can be removed.

Reply:
It crashes if I don't set any background resource. So I added empty drawable file which is now , updated you can check it.